### PR TITLE
feat: set client affinity and FTPS state

### PR DESCRIPTION
**Pull request by sebastianlolv**
### Checklist

- [x] I've updated both the `azurerm_linux_web_app` and `azurerm_windows_web_app` resources.

### CODE REVIEW

**Comment by sebastianlolv** [comment]
I am importing resources in a project, but with the current version changes these arguments when importing.

File: variables.tf [hunk 1]
```diff
@@ -82,6 +88,12 @@ variable "container_registry_managed_identity_client_id" {
   default     = null
 }
 
+variable "fpts_state" {
```
Comment (hknutsen) [inline variables.tf:91]: Small typo here 👀 

```suggestion
variable "ftps_state" {
```

File: main.tf [hunk 2]
```diff
@@ -137,6 +140,7 @@ resource "azurerm_windows_web_app" "this" {
     websockets_enabled                            = var.websockets_enabled
     container_registry_use_managed_identity       = var.container_registry_use_managed_identity
     container_registry_managed_identity_client_id = var.container_registry_managed_identity_client_id
+    ftps_state                                    = var.fpts_state
```
Comment (hknutsen) [inline main.tf:143]: Small typo here 👀

```suggestion
    ftps_state                                    = var.ftps_state
```

File: main.tf [hunk 3]
```diff
@@ -54,6 +55,7 @@ resource "azurerm_linux_web_app" "this" {
...
```
Comment (hknutsen) [inline main.tf:58]: Small typo here 👀

```suggestion
    ftps_state                                    = var.ftps_state
```

**Review by hknutsen — APPROVED** [review]
Looks good 🎉

**Comment by hknutsen** [comment]
See updated PR title 👀

**Pull request merged**